README.md

```diff
@@ -1,135 +1,170 @@
-# Diarizers
+# 🤗 Diarizers
 
-Diarizers is a library for fine-tuning [`pyannote`](https://github.com/pyannote/pyannote-audio/tree/develop) speaker diarisation models using Hugging Face tools. It provides scripts and tools to: 
-
-- Convert Speaker Diarization datasets into Hugging Face datasets compatible with the `diarizers` library. 
-- Fine-tune speaker diarization models using the Hugging Face `Trainer`. 
-- Test the models at inference time. 
-- Convert the fine-tuned models into a format compatible with `pyannote`.  
-
-> [!IMPORTANT]
-> For now, this library can only be used to fine-tune the [segmentation model](https://huggingface.co/pyannote/segmentation-3.0) from the [speaker diarization pipeline](https://huggingface.co/pyannote/speaker-diarization-3.1). 
-> Future work will aim to help optimise the hyperparameters of the entire pipeline. 
+🤗 Diarizers is a library for fine-tuning [`pyannote`](https://github.com/pyannote/pyannote-audio/tree/main) speaker 
+diarization models using the Hugging Face ecosystem. It can be used to improve performance on both English and multilingual 
+diarization datasets with simple example scripts, with as little as ten hours of labelled diarization data and just 5 minutes
+of GPU compute time.
 
 ## 📖 Quick Index
 * [Installation](#installation)
 * [Train](#train)
-* [Inference](#inference)
+* [Evaluation](#evaluation)
+* [Inference](#inference-with-pyannote)
 * [Results](#Results)
-* [Adding new datasets](#addingnewdatasets)
+* [Adding new datasets](#adding-new-datasets)
 * [Acknowledgements](#acknowledgements)
 * [Citation](#citation)
 
 ## Installation
 
-Diarizers has light dependencies and can be installed with the following lines: 
+First, clone the repository and install the dependencies:
 
 ```sh
-pip install git+https://github.com/huggingface/diarizers.git
-pip install diarizers[dev]
+git clone https://github.com/huggingface/diarizers.git
+cd diarizers
+pip install -e .
 ```
 
-You'll need to generate a [user access token](https://huggingface.co/docs/hub/en/security-tokens), and login with: 
```

Comment:
The pre-trained models are gated on the Hub with user terms-of-usage agreements. Have you considered how we're going to gate the fine-tuned models? We might need to add some custom logic to `push_to_hub` to ensure we gate the checkpoints in the same way the pre-trained ones are gated. This involves adding the following fields to the yaml section of the README: https://huggingface.co/pyannote/speaker-diarization-3.1/blob/0c6d72ac70c2dca2b11b236f5ca3d54d0c133109/README.md?code=true#L16-L19

Reply:
Ok, how can I add the custom logic to `push_to_hub` to do so ?

Reply:
We could also ask Hervé what he thinks of it!